lib/github.atd

```diff
@@ -590,7 +600,7 @@ type repo_issue_event = {
   ?label: base_label option;
   ?assignee: linked_user option;
   ?assigner: linked_user option;
-  ?milestone: milestone option;
+  ?milestone: repo_issue_milestone option;
```

Comment:
I've force pushed a split of the last commit into two. The `milestone` field is also part of the repository events, should this be updated as well?